packages/gatsby-source-wordpress-experimental/src/gatsby-node/on-pre-bootstrap/build-queries-from-introspection/index.js

```diff
@@ -103,7 +88,31 @@ export const buildNodeQueriesFromIntrospection = async (
     }
   }
 
-  //
-  // set the queries in our redux store
-  store.dispatch.introspection.setQueries(queries)
+  return queries
+}
+
+export const buildNodeQueriesFromIntrospection = async (
+  { introspection, schemaHasChanged = false },
+  helpers
+) => {
+  const QUERY_CACHE_KEY = `introspection-node-queries`
```

Comment:
I'm not super familiar with the cache layer here, but if I'm using this plugin to source from multiple WordPress sites, will this introspect from Site A and Site B and conflict in the cache? Or does `helpers.cache.get(QUERY_CACHE_KEY)` / `helpers.cache.set(QUERY_CACHE_KEY)` compensate for multiple WordPress (or non WordPress?) sources?

Reply:
@jasonbahl that's a great question! I haven't tested this plugin with multiple instances pointing at different WP sites. The cache is namespaced in the current plugin (`gatsby-source-wordpress-experimental`), so if there were multiple sites being used, this would be a problem for sure.

Reply:
Just scoped it to the current site url